src/components/Footer/Footer.jsx

```diff
@@ -5,14 +5,12 @@ import {
   FooterNav,
   VersionWrapper,
   CssToggle,
-  StyledSelect,
+  HelpLinksWrapper,
   HelpContainer,
 } from './Footer.styles'
 import { TextLink } from '../generic/links'
 import { useOnlineStatus } from '../../library/onlineStatusContext'
 import { versionNumber } from '../../version'
-import MermaidDocsEN from '../../docs/MERMAID-user-docs-EN-min.pdf'
-import MermaidDocsBIN from '../../docs/MERMAID-user-docs-ID-min.pdf'
 import OfflineHide from '../generic/OfflineHide'
```

Comment:
Jest didnt like importing these PDFs becasue it wanted to parse the contents of the file rather than grab a url, so I got rid of them in favour of just using a-tags.

Reply:
Help doc download should be QAed as well as the reference spreadsheet download in the header